device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-D48C8/BALANCED/buffers_defaults_t0.j2

```diff
@@ -8,10 +8,17 @@
     {%- endfor %}
 {%- endmacro %}
 
+{%- set ingress_lossless_pool_size = '32689152' %}
+{%- set egress_lossless_pool_size = '32340992' %}

```

Comment:
Minor question on this change: won't it cause the same problem in the non-dual tor case?

Reply:
non dualtor have same problem, but haven't completed verification for non-dualtor. commit dualtor's change first.